.github/workflows/codeql-analysis.yml

```diff
@@ -64,8 +64,6 @@ jobs:
         $LibraryTargetFrameworks=$LibraryUnitTestTargetFrameworks=$LibrarySamplesTargetFrameworks=$LibrarySamplesTargetFrameworksWindows="${{ matrix.framework }}"
         Switch($LibrarySamplesTargetFrameworks)
         {
```

Comment:
MINOR: This switch statement can be simplified, since we just need to add `-windows`.

Reply:
I was thinking the same. However, as we want to test Velocity more excessively on other platforms as well, I was thinking that this could come in handy while still making it easy to understand.